### PR TITLE
test: Cleanup integration test scripts

### PR DESCRIPTION
**Pull request by sayboras**
Please refer to individual commit for more details.

### CODE REVIEW

**Review by christarazi — APPROVED** [review]

File: .github/actions/unit-tests/prepare.sh [hunk 1]
```diff
@@ -45,4 +45,6 @@ install_clang
 
 export PATH="/usr/local/clang/bin:$PATH"
 
-go install github.com/mfridman/tparse@28967170dce4f9f13de77ec857f7aed4c4294a5f # v0.12.3 (main) with -progress
+# renovate: datasource=github-release-attachments depName=mfridman/tparse
```
Comment (tklauser) [inline .github/actions/unit-tests/prepare.sh:48]: I think you'll need to add this file to the `includePaths` and the respective `regexManagers.fileMatch` sections in `.github/renovate.json5` in order for renovate to be able to update this.

Comment (sayboras) [reply .github/actions/unit-tests/prepare.sh:48]: ah thanks, I always forgot about the above :+1:

File: .github/renovate.json5 [hunk 2]
```diff
@@ -511,5 +511,16 @@
         "# renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)\\s+.+_VERSION:=(?<currentValue>.*)\\s+.+_DIGEST:=(?<currentDigest>sha256:[a-f0-9]+)"
       ]
     },
+    {
+      "fileMatch": [
+        ".github/actions/unit-tests/prepare.sh",
```
Comment (tklauser) [inline .github/renovate.json5]: I think we could just reuse the rexisting entry at https://github.com/cilium/cilium/pull/31685/files#diff-20ab1190d08a53a3012bc191ed56205c8f0ffb8fa1715e9d36ecfd1d2ea8a790L450-L466 for this file as well (i.e. just add `.github/actions/unit-tests/prepare.sh` to on line 454. And then you'll need to add an entry for the file to the `includePaths` section at the top of `.github/renovate.json5` (or extend the glob patterns) as well.

**Review by tklauser — CHANGES_REQUESTED** [review]

**Pull request closed without merging**